de.metas.contracts/src/main/sql/postgresql/system/50-de.metas.contracts/5488820_sys_gh3718_AddExtensonTypeList.sql

```diff
@@ -0,0 +1,144 @@
+-- 2018-03-19T17:21:01.339
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,ColumnName,Created,CreatedBy,EntityType,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,543934,0,'ExtensionType',TO_TIMESTAMP('2018-03-19 17:21:01','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.contracts','Y','Extension Type','Vertrag autom. verlängern',TO_TIMESTAMP('2018-03-19 17:21:01','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 2018-03-19T17:21:01.353
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Element_Trl (AD_Language,AD_Element_ID, Description,Help,Name,PO_Description,PO_Help,PO_Name,PO_PrintName,PrintName, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Element_ID, t.Description,t.Help,t.Name,t.PO_Description,t.PO_Help,t.PO_Name,t.PO_PrintName,t.PrintName, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Element t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Element_ID=543934 AND NOT EXISTS (SELECT 1 FROM AD_Element_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Element_ID=t.AD_Element_ID)
+;
+
+-- 2018-03-19T17:27:44.481
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Reference (AD_Client_ID,AD_Org_ID,AD_Reference_ID,Created,CreatedBy,EntityType,IsActive,IsOrderByValue,Name,Updated,UpdatedBy,ValidationType) VALUES (0,0,540843,TO_TIMESTAMP('2018-03-19 17:27:44','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.contracts','Y','N','ExtensionType',TO_TIMESTAMP('2018-03-19 17:27:44','YYYY-MM-DD HH24:MI:SS'),100,'L')
+;
+
+-- 2018-03-19T17:27:44.493
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Reference_Trl (AD_Language,AD_Reference_ID, Description,Help,Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Reference_ID, t.Description,t.Help,t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Reference t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Reference_ID=540843 AND NOT EXISTS (SELECT 1 FROM AD_Reference_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Reference_ID=t.AD_Reference_ID)
+;
+
+-- 2018-03-19T17:36:43.631
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Ref_List (AD_Client_ID,AD_Org_ID,AD_Ref_List_ID,AD_Reference_ID,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value,ValueName) VALUES (0,0,541602,540843,TO_TIMESTAMP('2018-03-19 17:36:43','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.contracts','Y','Extend contract for all periods',TO_TIMESTAMP('2018-03-19 17:36:43','YYYY-MM-DD HH24:MI:SS'),100,'EA','ExtendAll')
+;
```

Comment:
pls describe better what it does
* that it's done on complete (right?)
* that there is a chain of "extends", until a transition record without this falg and a next conditions reference is reached